src/agent/messages.js

```diff
@@ -41,6 +43,7 @@ export const AGENT_TEMPLATES = {
   [AgentMessage.SPEECH_EVENT]: ({ spokenText }) => `Speech event: '${spokenText}'.`,
   [AgentMessage.NO_RUN_TEST_SETUP]: ({ referencePage }) =>
     `Test reference, ${referencePage}, does not have a Run Test Setup button.`,
+  [AgentMessage.DEBUG]: ({ msg }) => `[debug] ${msg}`,
```

Comment:
I couldn't figure out how to get the log to work without taking an object shaped like `{ msg }`  instead of just a string, but again, might back this change out entirely, it's not used in the final